features/FxA-54-push-on-status-change/README.md

```diff
@@ -30,6 +30,9 @@ As a concrete metric
 we can measure the number of calls to `/recovery_email/status`
 and check that it decreases as this feature rolls out.
 
+We will also add additional metrics to signal when Desktop verifies 
+via Push.
```

Comment:
Thanks @vladikoff!.  I'd like to hear more about the specific thing we're going to measure here, e.g. "number of status checks as a result of a push notifications, as a percentage of the number of push notifications sent", with the expectation that this will be a fairly high percentage.